ql/ql/src/queries/style/RedundantImport.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @name Redundant import
+ * @description An import that is redundant due to an earlier import
```

Comment:
FYI, I just noticed that the description ends with a '.' in all or most of javascript queries. This convention is not used in ql-for-ql.